collector/pkg/component/receiver/cgoreceiver/config.go

```diff
@@ -1,11 +1,24 @@
 package cgoreceiver
 
 type Config struct {
-	SubscribeInfo []SubEvent `mapstructure:"subscribe"`
+	SubscribeInfo     []SubEvent    `mapstructure:"subscribe"`
+	ProcessFilterInfo ProcessFilter `mapstructure:"process_filter"`
 }
 
 type SubEvent struct {
 	Category string            `mapstructure:"category"`
 	Name     string            `mapstructure:"name"`
 	Params   map[string]string `mapstructure:"params"`
 }
+
+type ProcessFilter struct {
+	Comms []string `mapstructure:"comms"`
+}
+
+func NewDefaultConfig() *Config {
```

Comment:
Could you help also put the fields of `SubscribeInfo` in the configuration file under the `NewDefaultConfig` method?

Reply:
I think so.